shortcircuit-api/src/test/resources/shortcircuit-results.csv

```diff
@@ -0,0 +1,6 @@
+Short circuit analysis
```

Comment:
I've put ; and dots on numbers to meet the CI requirements.
However, I am not sure that it should be like that...